docs/trouble_shooting.md

```diff
@@ -27,3 +27,25 @@ This guide (based on minikube and others should be similar) explains general inf
    ```
    # kubectl --kubeconfig minikube.kubeconfig log clusterapi-controller-0 -n ibmcloud-provider-system
    ```
+
+## Cannot create bootstrap cluster if you are using kind
+   
+1. Check if kind works well.
+
+   ```
+   # kind create cluster
+   Creating cluster "kind" ...
+   ✓ Ensuring node image (kindest/node:v1.14.2) 🖼
+   ✓ Preparing nodes 📦
+   ✓ Creating kubeadm config 📜
+   ✓ Starting control-plane 🕹️
+   ✓ Installing CNI 🔌
+   ✓ Installing StorageClass 💾
+   Cluster creation complete. You can now use the cluster with:
+
+   export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"
+   kubectl cluster-info
+   ```
+
+2. Check docker version.
+   kind v0.3.0 can work with docker 18.06.0 on ubuntu.
```

Comment:
Can you please add a link from the document of KIND for this? Do you mean KIND 0.3.0 will not work for docker <18.06 on Ubuntu?

Reply:
kind 0.3.0 do not work with docker 18.09.6 by test, and works after downgrade to 18.06.
No found kind's doc write what docker version to support. May bug for the latest docker version.

Reply:
Can you help open an issue in KIND and link the issue in your document?

Reply:
Also I think you only need to mention kind 0.3.0 does not work with docker 18.09.6, as in my MacOS, kind 0.3.0 works with docker 18.09.2

```
bogon:~ guangyaliu$ docker version
Client: Docker Engine - Community
 Version:           18.09.2
 API version:       1.39
 Go version:        go1.10.8
 Git commit:        6247962
 Built:             Sun Feb 10 04:12:39 2019
 OS/Arch:           darwin/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.2
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.6
  Git commit:       6247962
  Built:            Sun Feb 10 04:13:06 2019
  OS/Arch:          linux/amd64
  Experimental:     false
bogon:~ guangyaliu$ kind get clusters
kind
```

Reply:
@wanghh2000 sorry, I mean open an issue at https://github.com/kubernetes-sigs/kind/issues for some discussion from KIND experts.

Reply:
Appended issue on this PR.

Reply:
Issue: https://github.com/kubernetes-sigs/kind/issues/567 to check for Ubuntu. 
Let me check docker 18.09.02 on Ubuntu.